mypy_django_plugin/lib/helpers.py

```diff
@@ -136,7 +136,7 @@ def copy_method_to_another_class(
                 method_node.arguments[1:]):
             bound_arg_type = self.semanal_api.anal_type(arg_type, allow_placeholder=True)
             if bound_arg_type is None and not self.defer_till_next_iteration(reason='bound_arg_type is None'):
-                raise new_helpers.TypeInfoNotFound('of ' + arg_name + ' argument of ' + method_node.fullname)
+                raise new_helpers.TypeInfoNotFound('of ' + arg_name + ' argument of ' + method_node.fullname) # type: ignore
```

Comment:
Why do we need `type: ignore` here?

Reply:
Mypy was resolving `arg_name` as `None`, but now it's changed to:
```
error_msg = 'of {} argument of {}'.format(arg_name, method_node.fullname)
raise new_helpers.TypeInfoNotFound(error_msg)
```
and it works just fine

Reply:
👍